exo-go/src/app_config_helpers/index_test.go

```diff
@@ -0,0 +1,137 @@
+package appConfigHelpers_test
+
+import (
+	"os"
+	"path"
+
+	"github.com/Originate/exosphere/exo-go/src/app_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/types"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	appConfig types.AppConfig
+	services  types.Services
+)
+
+var _ = BeforeSuite(func() {
+	exocom := types.Dependency{Name: "exocom", Version: "0.22.1", Silent: true}
+	mongoConfig := types.DependencyConfig{DependencyEnvironment: map[string]string{"DB_NAME": "test-db"}}
+	mongo := types.Dependency{Name: "mongo", Version: "3.4.0", Config: mongoConfig}
+	dependencies := []types.Dependency{exocom, mongo}
+
+	todoService := types.ServiceData{Location: "./todo-service", NameSpace: "todo"}
+	externalService := types.ServiceData{DockerImage: "originate/test-web-server", Silent: true}
+	services = types.Services{
+		Private: map[string]types.ServiceData{"todo-service": todoService},
+		Public:  map[string]types.ServiceData{"external-service": externalService},
+	}
+
+	appConfig = types.AppConfig{
+		Name:         "test ",
+		Version:      "0.1.0",
+		Services:     services,
+		Dependencies: dependencies,
+	}
+})
+
+var _ = Describe("GetAppConfig", func() {
+	var appConfig types.AppConfig
+
+	It("should read application.yml successfully", func() {
+		appDir := path.Join("..", "..", "..", "exosphere-shared", "example-apps", "complex-setup-app")
+		if err := os.Chdir(appDir); err != nil {
+			panic(err)
+		}
+		var err error
+		appConfig, err = appConfigHelpers.GetAppConfig()
```

Comment:
I'd suggest updating this function to accept a `cwd` so that the tests don't need to change the current working directory.